Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
+        if #available(OSX 10.10, *) {
```

Comment:
Are you sure? Looking at both the Xcode project and the podspec, iOS 8.0 is listed as the deployment target. Am I looking in the wrong place?